tsconfig.json

```diff
@@ -11,17 +11,19 @@
     "declaration": true,
     "emitDeclarationOnly": true,
     "declarationDir": "types",
+    "outDir": "dist",
     "removeComments": true,
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "forceConsistentCasingInFileNames": true,
     "strict": true,
     "noImplicitAny": true,
     "strictNullChecks": true,
-    "typeRoots": [
-      "./types/index.d.ts",
-      "node_modules/@types"
-    ],
+    "skipLibCheck": true
+    // "typeRoots": [
+    //   "./types/index.d.ts",
+    //   "node_modules/@types"
+    // ],
```

Comment:
Not required?

Reply:
dont think so, it should automatically do it.